client/src/app/signup/signup.component.html

```diff
@@ -19,18 +19,18 @@
                     <div class="Signup_Body__Password_Wrapper">
                         <mat-form-field appearance="standard" [style.width.%]="100">
                             <mat-label>Enter Password</mat-label>
-                            <input matInput type="password" [(ngModel)]="credentials.password" #password="ngModel" name="password" id="password" required>
+                            <input matInput type="password" minlength="8" [(ngModel)]="credentials.password" #password="ngModel" name="password" id="password" required>
```

Comment:
Message will not be displayed for minlength. Use <mat-error> with ng if and capture the error with password.errors.minlength

Reply:
Done